File: kbe/src/client/unity3d/Method.cs
namespace KBEngine
{
  	using UnityEngine; 
	using System; 
	using System.Collections; 
	using System.Collections.Generic;
	
    public class Method 
    {
		public string name = "";
    	public UInt16 methodUtype = 0;
    	public Int16 aliasID = -1;
		public List<KBEDATATYPE_BASE> args = new List<KBEDATATYPE_BASE>();
		public System.Reflection.MethodInfo handler = null;
		
		public Method()
		{
			
		}
    }
    
} 

File: kbe/res/client/sdk_templates/ue4/Source/KBEnginePlugins/Private/Method.cpp

#include "Method.h"
#include "DataTypes.h"
#include "KBDebug.h"

Method::Method():
	name(TEXT("")),
	methodUtype(0),
	aliasID(-1),
	args(),
	pEntityDefMethodHandle(NULL)
{
}

Method::~Method()
{
	KBE_SAFE_RELEASE(pEntityDefMethodHandle);
}
